company-profiles/gft.md

```diff
@@ -0,0 +1,29 @@
+# GFT
+
+## Company blurb
+
+GFT Technologies SE (GFT) is a business and technology consultancy trusted by the world's leading financial institutions to solve their business challenges. GFT responds to the constant current regulatory change and continuously innovates to meet the demands of the digital revolution. GFT combines advice, creativity and technology with innovation and specialized knowledge of the financial sector to transform the businesses of its clients.
+
+## Company size
+
+GFT has +10.000 employees worldwide.
+
+## Remote status
+
+Full-remote and "work-from-home" policies are common across all regions where GFT has presence.
+
+## Region
+
+Asia, Europe, Latin America and North America
```

Comment:
The readme says "worldwide" - which is correct, this sentence or the readme (since this sentence doesn't include Australia/Oceania).

Reply:
The correct one will be the line under ## Region section. So we should modify the sentence under ## Company Size.

Let me push a commit fixing this. 😃